props/nested-components.md

```diff
@@ -0,0 +1,195 @@
+# Nested Components
+
+## Goals
+
+In order to increase the complexity of our React webapps, we should practice nesting components.
+
+Nested components share a relationship that is different from sibling components.
+
+We can apply a design pattern for components that will help us distinguish what our components do.
+
+## Format
+
+This lesson will take the following steps:
+
+1. Introduce two design patterns for components:
+   - presentational components
+   - container components
+1. Follow Sofia's journey on creating an attendance app, applying these design patterns
+
+## Nested Components
+
+When components are rendered, they can contain one or more elements or _other components_.
+
+For example, consider this definition of an `App` component:
+
+<!-- prettier-ignore-start -->
+```js
+function App() {
+  return (
+    <main>
+      <h1>Attendance</h1>
+      <ClassInfo></ClassInfo>
+      <StudentList></StudentList>
+    </main>
+  );
+}
+```
+<!-- prettier-ignore-end -->
+
+When an `App` component is rendered, it begins with a `<main>` element.
+
+The `App` component has two _nested_ components, or components rendered inside it:
+
+1. One instance of a `ClassInfo` component
+1. One instance of a `StudentList` component
+
+The `ClassInfo` and `StudentList` components are _siblings_ to each other.
+
+As React webapps increase in complexity, more and more components are defined and rendered.
+
+Deciding how to design a React webapp and its components can be challenging. We can learn about and apply two design patterns to help us:
+
+1. Presentational components
+1. Container components
+
+These are design patterns; these labels don't affect the syntax used to define them. The difference between these two kinds of components is the nature of their responsibilities, and how we use them.
+
+## Presentational Components
+
+Presentational components are components that are responsible for displaying data.
+
+Presentational components are not responsible for modifying data.
+
+## Container Components
+
+Container components are components responsible for:
+
+- Using presentation components
+- Holding data
+- Modifying and managing data
+- Sending data to presentation components
+
+Container components often have presentation components _nested_ inside them.
+
+## Example: `StudentList` and `Student` Components
+
+Sofia is a teacher creating an attendance app. Her app should display a list of students. At this moment, she starts with this implementation of a `StudentList` component in `src/components/StudentList.js`.
```

Comment:
I think I've commented on "at this moment" before as being a curious mixture of immediacy in a somewhat nebulous timeframe.
```suggestion
Sofia is a teacher creating an attendance app. Her app should display a list of students. Initially, she starts with this implementation of a `StudentList` component in `src/components/StudentList.js`.
```

Reply:
Bahhhh yes!! I think I've been struggling with timing things (I had to edit a whole lesson after I realized I wrote that Sofia _can do things_ instead of _is doing things_ hahaha. Ughh)

Thank you again!